### PR TITLE
Search user

### PR DESCRIPTION
**Pull request by shashikantmaurya0007**
feature:unfollow,search,debounce

### CODE REVIEW

**Review by vishalg8454 — APPROVED** [review]
Neat Code, everything looks good to be merged.

File: src/component/Search/SearchResult.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+import { FollowUnfollowCard } from "../../FollowUnfollowCard/FollowUnfollowCard";
+
+const SearchResult = ({ searchInput, allUsers }) => {
+  const isValidSearch = Boolean(searchInput.trim());
+
+  let searchedUser = [];
+  if (isValidSearch) {
+    searchedUser = allUsers.filter((ele) =>
+      ele.username.includes(searchInput.trim())
```
Comment (MaruthiPothuganti) [inline src/component/Search/SearchResult.jsx]: Missed handling the edge case of Alphabet casing. PLease fix it.

**Review by MaruthiPothuganti — APPROVED** [review]
Recommended a fix. Everything else looks good and PR can be merged.

**Review by sharmayatendra — APPROVED** [review]
code looks clean 
design is consistent
file structure is also good

File: src/component/Search/SearchResult.jsx (hunk 1 above)
Comment (shashikantmaurya0007) [reply src/component/Search/SearchResult.jsx]: yes fixed it thanks for pointing this out

**Pull request merged**